eslint.config.mjs

```diff
@@ -0,0 +1,15 @@
+import eslintConfigInternxt from '@internxt/eslint-config-internxt';
+
+export default [
+    {
+        ignores: ['dist', 'tmp', 'scripts'],
+    },
+    ...eslintConfigInternxt,
+    {
+        rules: {
+            'no-console': 'off',
+            'max-len': 'off',
+            '@typescript-eslint/no-empty-function': 'off',
+        }
```

Comment:
This warnings were already disabled [(im not adding them really).](https://github.com/internxt/cli/blob/041f06d8b61a5ac0ef6d42a81d0d155d70ad00f7/.eslintrc.json#L5) I think they were added because of this long lines: https://github.com/internxt/cli/blob/041f06d8b61a5ac0ef6d42a81d0d155d70ad00f7/test/webdav/handlers/PROPFIND.handler.test.ts#L80

Reply:
I have finally disabled it and added some exceptions!